knowledge-base/common-component-requires-telerikrootcomponent.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Telerik Component Requires a TelerikRootComponent
+description: Learn how to fix the exception about a Telerik component on the requested view requires a TelerikRootComponent in MainLayout or a parent component.
```

Comment:
```suggestion
description: Learn how to fix the exception "А Telerik component on the requested view requires a TelerikRootComponent in MainLayout or a parent component".
```

Reply:
Quotation marks in the page meta information will break the resulting HTML.